### PR TITLE
Assignment4-AndreiSaavedra

### PR DESCRIPTION
**Pull request by saavedrAndrei**

### CODE REVIEW

File: Assignment4/saavedrAndrei-117032611/task06.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+Original file is located at
+    https://colab.research.google.com/drive/1ZXG5LLlmp3oFKdUfPxRIhYIIE7AEiIcx
+
+**saavedrAndrei - Andrei Saavedra**
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/saavedrAndrei-117032611/task06.py:14]: won't work in python. This is notebook magic

File: Assignment4/saavedrAndrei-117032611/task06.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0/")
+
+upm = g.add((ns.UPM, RDF.type, ns.University))
+
+from rdflib import FOAF
+
+ns = Namespace("http://somewhere#")
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0/")
+
+g.add((ns.JohnSmith, vcard.work, ns.UPM))
```
Comment (dgarijo) [inline Assignment4/saavedrAndrei-117032611/task06.py:92]: you are defining the work property in vcard, but it does not exist. Create your own

File: Assignment4/saavedrAndrei-117032611/task07.py [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    SELECT ?s 
+    WHERE {
+      ?s rdf:type ns:Person .
+    }
+  }
+  UNION
+  {
+    SELECT ?s
+    WHERE {
+      ?subclass rdfs:subClassOf ns:Person .
```
Comment (dgarijo) [inline Assignment4/saavedrAndrei-117032611/task07.py:80]: use * to get all subclass hierarchy

**Review by dgarijo — CHANGES_REQUESTED** [review]

**Comment by saavedrAndrei** [comment]
Assignment4-Corrected

**Review by dgarijo — APPROVED** [review]

**Pull request merged**